src/verso-manual/VersoManual/Html/Style.lean

```diff
@@ -74,46 +86,35 @@ dd > p:first-child {
 pre, code {
     font-family: var(--verso-code-font-family);
     font-variant-ligatures: none;
+    overflow-x: auto;
 }
 
 /******** Page Layout ********/
 
-.with-toc {
-    display: grid;
-    grid-template-columns: min-content auto;
-    grid-template-rows: auto;
-    grid-template-areas:
-        "toc text";
-    height: 100vh;
-    overflow: hidden;
-}
-
 .with-toc #toc {
-    grid-area: toc;
-    height: 100vh;
-}
-
-.with-toc #top-menu {
-    grid-area: toggle;
+    position: fixed;
+    z-index: 10;
 }
 
-.with-toc #toc {
-    overflow-y: auto;
+/** Non-mobile **/
+@media screen and (min-width: 701px) {
```

Comment:
I would not use both min-width and max-width across a stylesheet. In general, you either go mobile first or desktop first, with mobile first being the most common because something like 80% of websurfing is done on mobile.

But this page is built desktop-first, which makes sense since you'll mostly be browsing it while writing lean on your desktop machine. So let the default be for desktop and override the things you need to override for mobile.

Reply:
Looked it up. 60% :)

Reply:
Yeah I'm curious too. I know that 1/2 pixels are a thing on retina screens, so in this specific case you would get neither option at 700.5 px, but you can 'solve' that with `min-width: 700.01px`, though that's not satisfying.

For me I think of the stylesheet as being for the most-used platform, with tweaks to make it pretty on other platforms too. It reveals what the author of the sheet was thinking. If you make changes to your 'main' platform, you probably want most of those changes on the other platforms too (colors, font styles, the feel of the website). But I'm just theorizing. I also think there is value in 'just doing it like we always do', especially if you're likely to hire another consultant for this in the future.